extensions/jms/src/test/resources/config/activemq/activemq-default-no-caching.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:jmsn="http://www.mulesoft.org/schema/mule/jmsn"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+      http://www.mulesoft.org/schema/mule/jmsn http://www.mulesoft.org/schema/mule/jmsn/current/mule-jmsn.xsd">
+
+
+    <jmsn:config name="config">
+        <jmsn:active-mq-connection>
+            <jmsn:caching-strategy>
```

Comment:
it's the wrapper element because we have a Subtype mapping of: 
CachingStrategy -> DefaultCaching and NoCaching